### PR TITLE
remove checking of satellite id in QC

### PR DESCRIPTION
**Pull request by haydenlj**
## Description

*(Instructions: replace text in this and all sections with your own text)*

Provide a detailed description of what this PR does.

What problem does it fix? What new capability does it add?

[See the JEDI Documentation](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/pullrequest.html) for further information.

### Issue(s) addressed

Link the issues to be closed with this PR
- fixes #<issue_number>

If this does not close an existing issue, then [be sure to add a Pipeline, Label, Estimate, Assignees, and Epic](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/issues.html)

## Acceptance Criteria (Definition of Done)

What does it mean for this PR to be finished?

## Dependencies

If there are PRs that need to be merged before or along with this one, please add "waiting for another PR" label and list the dependencies in the other repositories (example below). Note that the branches in the other repositories should have matching names for the automated tests to pass.
Waiting on the following PRs:
- [ ] waiting on JCSDA/eckit/pull/<pr_number>
- [ ] waiting on JCSDA/atlas/pull/<pr_number>

## Impact

Please list the other repositories (if any) that this PR will require changes in (example below)

- [ ] saber
- [ ] ufo
- [ ] ...

### CODE REVIEW

**Review by hailingz — APPROVED** [review]
looks good!

File: src/gnssro/gnssro_bufr2ioda.f90 [hunk 1]
```diff
@@ -194,34 +194,26 @@ program gnssro_bufr2ioda2
...
-               if(ibit(i)== 6) then  ! refractivity
-                  refflag = 1
-                  exit
-               endif
-           enddo
-         endif
+     call upftbv(lnbufr,nemo,qfro,mxib,ibit,nib)
+     if(nib > 0) then
+        do i = 1, nib
+            if(ibit(i)== 5) then  ! bending angle
```
Comment (huishao-r) [inline src/gnssro/gnssro_bufr2ioda.f90:207]: The python script checks bit 1, 4, 5 but Fortran code here only checks 5, which might cause discrepancy?

**Review by BenjaminRuston — APPROVED** [review]

File: src/gnssro/gnssro_bufr2ioda.f90 (hunk 1 above)
Comment (huishao-r) [reply src/gnssro/gnssro_bufr2ioda.f90:207]: @haydenlj Was this fixed?

Comment (huishao-r) [reply src/gnssro/gnssro_bufr2ioda.f90:207]: I was talking about Ibit=5, which is for the bending angle. I was talking about matching the python code checks
` if i_non_nominal != 0 or i_phase_non_nominal != 0 or i_bang_non_nominal != 0:
        return {}` 
in the python converter. The code here only checks  i_bang_non_nominal (ibit 5), but not the other two

Comment (haydenlj) [reply src/gnssro/gnssro_bufr2ioda.f90]: I understand now, sorry. It's fixed

Comment (srherbener) [reply src/gnssro/gnssro_bufr2ioda.f90]: @huishao-r are you good with these updates, ie is it okay to merge? Thanks!

**Pull request merged**